app/backend/src/clj/teet/project/project_commands.clj

```diff
@@ -28,14 +30,23 @@
               :thk.project/custom-end-m]
           project-eid))
 
+(defn update-vektorio-project-name? [db project-id project-name]
+  (let [vektorio-enabled? (environment/feature-enabled? :vektorio)
+        vektorio-config (environment/config-value :vektorio)]
+    (when project-id
+          (log/debug "Updating project name in vektorio? " (some? vektorio-enabled?))
+      (when vektorio-enabled?
+            (vektorio/update-project-in-vektorio! db vektorio-config project-id project-name)))))
+
 (defcommand :thk.project/update
   {:doc "Edit project basic info"
    :context {:keys [conn db user]}
    :payload {id :thk.project/id :as project-form}
    :project-id [:thk.project/id id]
    :authorization {:project/update-info {:eid [:thk.project/id id]
                                          :link :thk.project/owner}}}
-  (let [{db-before :db-before
+  (if (some? (update-vektorio-project-name? db [:thk.project/id id] (:thk.project/project-name project-form)))
```

Comment:
So does this whole `thk.project/update` fail when the vektorio feature folag is not enabled?